docs/sources/alerting/alerting-rules/_index.md

```diff
@@ -5,18 +5,18 @@ aliases:
   - unified-alerting/alerting-rules/
   - ./create-alerts/
 canonical: https://grafana.com/docs/grafana/latest/alerting/alerting-rules/
-description: Create and manage alert rules
+description: Configure alert rules
 labels:
   products:
     - cloud
     - enterprise
     - oss
-menuTitle: Create and manage alert rules
-title: Create and manage alert rules
+menuTitle: Configure alert rules
```

Comment:
nit: this can be removed since it's the same as the title.

Reply:
true that

Reply:
done